src/com/testpro/hw4/Car.java

```diff
@@ -1,8 +1,62 @@
 package com.testpro.hw4;
 
-public class Car {
+class Car {
 
-    public static void main(String[] args){
 
+
+    private String Country_Of_Car;
```

Comment:
variable names should start with small letter and use camelCase without underlines